admin-dev/themes/default/sass/partials/_multistore.sass

```diff
@@ -34,37 +34,56 @@
 
 #header_shop
   .dropdown-menu
-    min-width: 210px
-    list-style: none
+    margin: 0
+    padding: 0
+    border-radius: 0
+    min-width: 17.75rem !important
+    box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.1)
+    border: 1px solid #bbcdd2
+    top: 40px
+    right: -100px !important
```

Comment:
!important is bad, are you sure there is no other way?

Reply:
I agree, either way don't worry too much about the default theme as it will die with 1.7

In this case, I think it was "important", because it was needed to to move an already aligned floating menu.